lib/src/button/button.component.spec.ts

```diff
@@ -30,8 +31,8 @@ class TestAppReference {
     constructor() {
         this.fixture = TestBed.createComponent(TestAppComponent);
         this.testAppComponent = this.fixture.debugElement.componentInstance;
-        this.buttonDebugElement = this.fixture.debugElement.query(By.css('button'));
-        this.aDebugElement = this.fixture.debugElement.query(By.css('a'));
+        this.buttonDebugElement = this.fixture.debugElement.query(By.css('button') as any) as any;
```

Comment:
I'm guessing these `as any`s have to do with the no implicit any rule?

Reply:
This was because I was getting this error when running the tests:

>  02 02 2018 16:39:34.737:ERROR [karma]: Error: lib/src/button/button.component.spec.ts(34,67): > error TS2345: Argument of type 'Predicate<DebugElement>' is not assignable to parameter of type 'Predicate<DebugElement>'.